examples/website/src/shared/route/gqlFragments.ts

```diff
@@ -294,8 +294,16 @@ export const linkPageBreakBlockDataFragment = gql`
   fragment LinkPageBreakBlockData on LinkPageBreakBlock {
     __typename
     text
+    htmlText
     linkURL
     linkText
+    hideButton
```

Comment:
aren't you missing the `templateOption` here?

Reply:
yes, corrected